test/reducedim.jl

```diff
@@ -58,6 +64,7 @@ safe_minabs(A::Array{T}, region) where {T} = safe_mapslices(minimum, abs.(A), re
     @test @inferred(sum(abs2, Areduc, dims=region)) ≈ safe_sumabs2(Areduc, region)
     @test @inferred(maximum(abs, Areduc, dims=region)) ≈ safe_maxabs(Areduc, region)
     @test @inferred(minimum(abs, Areduc, dims=region)) ≈ safe_minabs(Areduc, region)
+    @test @inferred(count(Breduc, dims=region)) ≈ safe_count(Breduc, region)
```

Comment:
Is there a test for `count(f, A; dims=...)`?

Reply:
good catch! I added some missing tests